src/components/carsoulSection/highlightSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function HighLight(props) {
+  return (
+    <div className="buttomText bg-black  pt-1 pt-md-2 text-white font-bold text-center  highlight ">
+      {props.text}
+    </div>
+  );
+}
+HighLight.propTypes = {
+  text: PropTypes.string,
+};
+export default HighLight;
```

Comment:
the file name as well as the function name dose not make sense 
this should be called **highlightedSectionText** or **highlightedSectionLabel** or any thing else you see it fit
what do you think?